partials/_create.scss

```diff
@@ -0,0 +1,40 @@
+.create-list {
+    width: 100%;
+    text-align: center;
+    font-family: $font-regular;
+    font-size: 14px;
+    line-height: 24px;
+
+    &__header {
+        margin: 24px 0px 32px 0px;
+    }
+
+    &__chat {
+        padding: 24px 0px 96px;
+        width: 347px;
+        margin: auto;
+    }
+
+    &-btn {
+        width: 142px;
+        margin: auto;
+    }
+}
+
+.underline {
+    text-decoration: underline;
+    cursor: pointer;
+
+    &:hover {
+        color: $tws_coral;
+    }
+}
+
+.bow-large {
+    height: 30px;
+    width: 58px;
```

Comment:
if it's an image, you don't have to specify both these values, just one of them is enough, the second will adjust - it's not an error, it's just not necessary

Reply:
Noted, thank you